docs/src/structure-factor.md

```diff
@@ -81,28 +75,27 @@ available after the calculation has completed.
 3. `nω`: Determines the number of energy bins to resolve. A larger number will
    require more calculation time.
 
-Upon constructing `DynamicStructureFactor`, classical spin dynamics will be
-performed, and spin correlation data will be accumulated into $𝒮^{αβ}(𝐪,ω)$.
-The input `sys` must be a spin configuration in good thermal equilibrium, e.g.,
-using the continuous [`Langevin`](@ref) dynamics or using single spin flip
-trials with [`LocalSampler`](@ref). The statistical quality of the
-$𝒮^{αβ}(𝐪,ω)$ can be improved by generating a decorrelated spin configuration
-in `sys`, and then calling [`add_sample!`](@ref).
+A sample may be added by calling `add_sample!(sc, sys)`. The input `sys` must be
+a spin configuration in good thermal equilibrium, e.g., using the continuous
+[`Langevin`](@ref) dynamics or using single spin flip trials with
+[`LocalSampler`](@ref). The statistical quality of the $𝒮^{αβ}(𝐪,ω)$ can be
+improved by repeatedly generating decorrelated spin configurations in `sys` and
+calling `add_sample!` on each configuration.
 
 The outline of typical use case might look like this:
 ```
-# Make a `StructureFactor` and calculate an initial sample
-sf = DynamicStructureFactor(sys; Δt=0.05, ωmax=10.0, nω=100) 
+# Make a `SampledCorrelations` and calculate an initial sample
+sc = SampledCorrelations(sys; Δt=0.05, ωmax=10.0, nω=100) 
```

Comment:
This no longer has an initial sample as suggested by the comment

Reply:
Good catch. Thanks.